src/components/Database/Database.tsx

```diff
@@ -58,18 +74,35 @@ export const Database: React.FC<Props> = ({ config, namespace, path }) => {
 
   return (
     <div className="Database-Database">
-      {ref ? (
-        <>
+      {ref && api ? (
+        <ApiProvider value={api}>
           <InteractiveBreadCrumbBar
             base={urlBase}
             path={path || '/'}
             inputPrefix={getPrefix(ref)}
             onNavigate={handleNavigate}
-          />
+          >
+            <SimpleMenu
+              handle={<IconButton icon="more_vert" label="Open menu" />}
+              renderToPortal
```

Comment:
Looking at the API, this seems like a prop for SimpleMenuSurface, not SimpleMenu. Could you confirm you need this?

Reply:
It's needed otherwise this wraps:

<img width="265" alt="Screen Shot 2020-05-07 at 10 23 33 AM" src="https://user-images.githubusercontent.com/702990/81325165-d4c7ce80-904c-11ea-9cfe-ea576be629ba.png">